package.json

```diff
@@ -37,7 +37,8 @@
     "rimraf": "~2.5.0",
     "tar": "~2.2.1",
     "tv4": "~1.2.7",
-    "swagger-tools": "~0.9.11"
+    "swagger-tools": "~0.9.11",
+    "moment": "~2.11.1"
```

Comment:
If could, please write these module name in alphabet order, because our previous modules are all defined in such a convention. Of course, if you use following command to add the module, it will be automatically added by alphabetic order:
```
npm install swagger-tools --save
```